app/routes/products.$productSlug/route.tsx

```diff
@@ -87,24 +110,28 @@ export default function ProductDetailsPage() {
                     <UnsafeRichText className={styles.description}>{product.description}</UnsafeRichText>
                 )}
 
-                {product.productOptions?.map((option) => (
-                    <ProductOption
-                        key={option.name}
-                        error={
-                            addToCartAttempted && selectedOptions[option.name!] === undefined
-                                ? `Select ${option.name}`
-                                : undefined
-                        }
-                        option={option}
-                        selectedValue={selectedOptions[option.name!]}
-                        onChange={(value) =>
-                            setSelectedOptions((prev) => ({
-                                ...prev,
-                                [option.name!]: value,
-                            }))
-                        }
-                    />
-                ))}
+                {product.productOptions && product.productOptions.length > 0 && (
+                    <div className={styles.productVariants}>
```

Comment:
we have 2 terms here - productOptions and productVariants. Both mean same thing, but have different names. So let't unite them

Reply:
I'd like to think so as well :) In reality these a different things in `Product` entity of wix e-com platform. For example `size: 'small' | 'medium' | 'large'` and `color: 'black' | 'white'` are two separate options, at the same time the product with such options has 6 variants: `small-white`, `small-black`, `medium-white`, etc.

Reply:
ok. agree.
but then why do you render array of ProductOption inside the div.productVariants ? 
it seems I don't understand something here

Reply:
👍  changing this.. the className was added before I fully understood what is variant and what is option